test/TruffleExtensionAdapter.test.ts

```diff
@@ -37,4 +40,12 @@ describe("TruffleExtensionAdapter", () => {
     // Assert
     assert.strictEqual(deployContractsMock.calledOnce, true, "TruffleCommands.deployContracts should be called once");
   });
+
+  it("exec method should call truffleCommands.execScript", async () => {
+    // Act
+    await truffleExtensionAdapter.execScript(Uri.file("./test.js"));
+
+    // Assert
+    assert.strictEqual(execScriptMock.calledOnce, true, "TruffleCommands.execScript should be called once");
```

Comment:
similarly should this test mock something lower down and return a true/false on the success/failure of the exec call?

Reply:
possibly 🙃 do you mean whether the script actually executes (irrespective of its success / failure)? i feel this is more of a QOL of update and whether the script actually succeeds / fails doesn't matter too much as long as the output is accessible (as per the above)...it would be awesome if it auto-open to the output channel though